### PR TITLE
Fixed builds based on vcpkg

### PR DESCRIPTION
**Pull request by JohanMabille**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -300,9 +300,9 @@ jobs:
 
           text = f.read_text()
 
-          text = text.replace("b309799e5a9d248ef66eaf11a0bd21bf4e8b9bd5c677c627ec83fa760ce9f0b54ddf1b62cbb436e641fbbde71e3b61cb71ff541d866f8ca7717a3a0dbeb00ebf",
-                       "a202ddaefa93d8a4b15431dc514e3a6200c47275c5a0027c09cc32b28bc079b1b9a93d5ef65adafdc9aba5f76a42f3303b1492106ddf72e67f1801ebfe6d02cc")
-          text = text.replace("https://repo.msys2.org/msys/x86_64/libtool-2.4.6-9-x86_64.pkg.tar.xz", "https://repo.msys2.org/msys/x86_64/libtool-2.4.7-3-x86_64.pkg.tar.zst")
+          text = text.replace("fdd86f4ffa6e274d6fef1676a4987971b1f2e1ec556eee947adcb4240dc562180afc4914c2bdecba284012967d3d3cf4d1a392f798a3b32a3668d6678a86e8d3", "fbdcf2572d242b14ef3b39f29a6119ee58705bad651c9da48ffd11e80637e8d767d20ed5d562f67d92eecd01f7fc3bc351af9d4f84fb9b321d2a9aff858b3619")
```
Comment (Klaim) [inline .github/workflows/main.yml:303]: 🥲

**Review by Klaim — APPROVED** [review]
Welp.

File: .github/workflows/main.yml (hunk 1 above)
Comment (JohanMabille) [reply .github/workflows/main.yml:303]: Yes I don't know why the vcpkg index is not up to date, there are dozens of PR opened to fix it.

**Comment by SylvainCorlay** [comment]
Hopefully we will get rid of vcpkg altogether at some point.

**Pull request merged**